bridges/primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -57,6 +57,12 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(constants::WEIGHT_REF_TI
 	.saturating_div(2)
 	.set_proof_size(polkadot_primitives::MAX_POV_SIZE as u64);
 
+/// We allow for 2 seconds of compute with a 6 second average block.
+const MAXIMUM_BLOCK_WEIGHT_FOR_ASYNC_BACKING: Weight = Weight::from_parts(
+	constants::WEIGHT_REF_TIME_PER_SECOND.saturating_mul(2),
+	polkadot_primitives::MAX_POV_SIZE as u64,
+);
```

Comment:
Why do we still have this extra const?

Reply:
Forumlated differently, why do we need `BlockWeightsForAsyncBacking` and `BlockWeights`?

Reply:
It is used by our relayer (lives in a separate repo still) and now we have: (1) testnet bridge hubs, which are using async backing, so we'll be using this constant for those chains (2) production bridge hubs (Kusama and Polkadot), where we still need to this without-async-backing version until those are migrated to async backing too

Reply:
Ahh okay, I forgot this again.